mtags/src/main/scala/scala/meta/internal/mtags/ListFiles.scala

```diff
@@ -1,50 +1,21 @@
 package scala.meta.internal.mtags
 
-import java.nio.file.FileVisitResult
 import java.nio.file.Files
-import java.nio.file.NoSuchFileException
-import java.nio.file.Path
-import java.nio.file.SimpleFileVisitor
-import java.nio.file.attribute.BasicFileAttributes
 import scala.meta.io.AbsolutePath
-import scala.collection.mutable.ArrayBuffer
+import scala.collection.mutable
 
 object ListFiles {
-  def foreach(root: AbsolutePath)(fn: AbsolutePath => Unit): Unit = {
-    ListFiles(root).foreach(fn)
+  def apply(root: AbsolutePath): mutable.ArrayBuffer[AbsolutePath] = {
```

Comment:
this should be in an extension method for `AbsolutePath`, like `list` or `children`